Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0/MIT"
 repository = "https://github.com/gimli-rs/object"
 
 [dependencies]
-goblin = "0.0.12"
+goblin = { git = "https://github.com/m4b/goblin.git", rev = "76e8e0df5c6401bec274b1e6c561386106a8b6e2" }
```

Comment:
I'm assuming this is temporary, waiting on an upstream PR to merge?

Reply:
Yeah, my PRs have been merged, but need a goblin release with it. cc @m4b, no rush though.

Reply:
I wanted to get to this tonight but I'm too tired :face_with_head_bandage: I will try tomorrow...

Reply:
@philipc done